locales/sv_SE/pcsx2_Iconized.po

```diff
@@ -348,19 +348,7 @@ msgstr ""
 msgid ""
 "Off: Disables temporary aspect ratio switch.\n"
 "\n"
-"4:3: Temporarily switch to a 4:3 aspect ratio while an FMV plays to "
-"correctly display an 4:3 FMV.\n"
-"\n"
-"16:9: Temporarily switch to a 16:9 aspect ratio while an FMV plays to "
-"correctly display a widescreen 16:9 FMV."
```

Comment:
Hmm. Seems it doesn't like strings being split up into multiple lines. Needs investigating.